### PR TITLE
Replace Trackablescrollview with Trackablescroller across the

### PR DESCRIPTION
**Pull request by pantelisss**
## **Why?**
In a previous PR, we developed a new component `TrackableScroller` to track the scrolling offset of a scroller in a better way.
The point of this PR is to replace the old component with the new one
### **How?**
Replaced every scroller across the app and deleted the old component (`TrackableScroller`) along with the unnecessary dependencies (`PullableScrollView`)
### **Testing**
Ensure everything looks as expected in the following screens 
- Home
- Profile tab
- Every tab of station details
- Reward details
- Rewards timeline
- Boost view
- Explorer stations list
- My wallet
- Station settings
- History data
- Net stats
### **Additional context**
fixes fe-1348

## Release Notes

- **New Features**
	- Introduced `TrackableScrollOffsetObject` to manage scroll offsets.
	- Added `didEndDraggingAction` closure for improved user interaction feedback.

- **UI Enhancements**
	- Replaced `TrackableScrollView` with `TrackableScroller` across multiple views, improving scrolling behavior and visual presentation.
	- Added `.scrollIndicators(.hidden)` to enhance UI aesthetics.

- **Bug Fixes**
	- Improved error handling and state management in various view models.

These updates streamline user interactions and enhance the overall experience within the application.

### CODE REVIEW

File: PresentationLayer/UIComponents/Screens/ConnectWallet/MyWalletView.swift [hunk 1]
```diff
@@ -19,7 +19,7 @@ struct MyWalletView: View {
             Color(colorEnum: .bg).ignoresSafeArea()
 
             ZStack {
-                TrackableScrollView(offsetObject: viewModel.trackableObject) {
+                ScrollView {
```
Comment (PavlosTze) [inline PresentationLayer/UIComponents/Screens/ConnectWallet/MyWalletView.swift:22]: Not `TrackableScoller`?

File: PresentationLayer/UIComponents/Screens/AppUpdate/AppUpdateView.swift [hunk 2]
```diff
@@ -29,7 +29,7 @@ private struct ContentView: View {
 				.ignoresSafeArea()
 
 			VStack(spacing: CGFloat(.defaultSpacing)) {
-				TrackableScrollView {
+				ScrollView {
```
Comment (PavlosTze) [inline PresentationLayer/UIComponents/Screens/AppUpdate/AppUpdateView.swift:32]: Not `TrackableScoller`?

File: PresentationLayer/UIComponents/Screens/ExplorerStationsList/ExplorerStationsListView.swift [hunk 3]
```diff
@@ -51,7 +51,7 @@ private struct ContentView: View {
 					.wxmShadow()
 					.zIndex(1)
 
-                TrackableScrollView {
+                ScrollView {
```
Comment (PavlosTze) [inline PresentationLayer/UIComponents/Screens/ExplorerStationsList/ExplorerStationsListView.swift:54]: Not `TrackableScoller`?

**Review by PavlosTze — COMMENTED** [review]
@pantelisss Everything looks good but before approving it a small question, has the scoller changed in Forecast Details as well? Because I couldn't find it in the changed files.

I also noticed that in some occassions you replaced `TrackableScrollView` with `ScrollView` and in others with `TrackableScroller`, what's their difference?

**Comment by pantelisss** [comment]
> @pantelisss Everything looks good but before approving it a small question, has the scoller changed in Forecast Details as well? Because I couldn't find it in the changed files.
> 
> I also noticed that in some occassions you replaced `TrackableScrollView` with `ScrollView` and in others with `TrackableScroller`, what's their difference?

If there is no need for pull to refresh, we use the native `ScrollView`. In every other case (track offset and pull to refresh), we use the `TrackableScroller`.

**Pull request merged**